core/src/com/unciv/logic/automation/ConstructionAutomation.kt

```diff
@@ -187,6 +186,11 @@ class ConstructionAutomation(val cityConstructions: CityConstructions){
         if (preferredVictoryType == VictoryType.Cultural
                 && wonder.name in listOf("Sistine Chapel", "Eiffel Tower", "Cristo Redentor", "Neuschwanstein", "Sydney Opera House"))
             return 3f
+        // Only start building if we are the city that would complete it the soonest
+        if (wonder.uniques.contains("Wins the game when built") && cityInfo == civInfo.cities.minByOrNull { 
+                it.cityConstructions.getRemainingWork(it.cityConstructions.getCurrentConstruction().name) / it.cityStats.currentCityStats.production.roundToInt() + wonder.cost / it.cityStats.currentCityStats.production 
```

Comment:
CityConstructions contains a function of getTurnsToCompletion or something of that sort